src/argus_htmx/templates/htmx/incidents/_incidents_refresh_info.html

```diff
@@ -1,22 +1,30 @@
-<dl id="table-refresh-info" class="stats stats-horizontal shadow leading-none overflow-x-auto font-medium bg-neutral text-neutral-content">
-    <div class="stat py-2">
-        <dt class="stat-title text-neutral-content/80">Total, all time</dt>
-        <dd class="stat-value text-2xl text-base"> {{ count }}</dd>
-    </div>
-    <div class="stat py-2">
-        <dt class="stat-title text-neutral-content/80">After filtering</dt>
-        <dd class="stat-value text-2xl text-base"> {{ filtered_count }}</dd>
-    </div>
-    <div class="stat py-2">
-        <dt class="stat-title text-neutral-content/80">Per page</dt>
-        <dd class="stat-value text-2xl text-base">{{ per_page }}</dd>
-    </div>
-    <div class="stat py-2">
-        <dt class="stat-title text-neutral-content/80">Last refreshed</dt>
-        <dd class="stat-value text-2xl text-base">{{ last_refreshed|date:datetime_format|default:"?" }}</dd>
-    </div>
-    <div class="stat py-2">
-        <dt class="stat-title text-neutral-content/80">Updating every</dt>
-        <dd class="stat-value text-2xl text-base">{{ update_interval|default:"?" }}s</dd>
-    </div>
-</dl>
+<form hx-get="{% url 'htmx:incident-list' %}" id="table-refresh-info" hx-trigger="change" hx-target="#table" hx-swap="outerHTML">
+    <dl class="stats stats-horizontal shadow leading-none overflow-x-auto font-medium bg-neutral text-neutral-content">
+        <div class="stat py-2">
+            <dt class="stat-title text-neutral-content/80">Total, all time</dt>
+            <dd class="stat-value text-2xl text-base"> {{ count }}</dd>
+        </div>
+        <div class="stat py-2">
+            <dt class="stat-title text-neutral-content/80">After filtering</dt>
+            <dd class="stat-value text-2xl text-base"> {{ filtered_count }}</dd>
+        </div>
+        <div class="stat py-2">
+            <dt class="stat-title text-neutral-content/80">Per page</dt>
+            <dd class="stat-value text-2xl text-primary">
+                <select class="select select-xs" name="page_size">
+                    {% for ps in all_page_sizes %}
+                        <option value="{{ ps }}" {% if ps == page_size %}selected{% endif %}>{{ ps }}</option>
+                    {% endfor %}
+                </select>
```

Comment:
This could be encapsulated in a django form.

(Really need to move up to Django 5.1 for more form goodness first though.)

Reply:
Sure, but I find it sometimes hard to customize django forms. And since it was one select, it seemed ok to not do it as a form.

but feel free to change :)

Reply:
Had it been before the nordunet conference I would have said "after-demo". No rush :)